@ndlib/gatsby-theme-marble/src/components/Shared/Seo/helpers.js

```diff
@@ -27,7 +27,7 @@ export const getFieldValue = (field, label, data) => {
   const metaField = typy(data, 'marbleItem.metadata').safeArray.find(md => {
     return md.label === label
   })
-  return field || typy(metaField, 'value[0]').safeString || null
+  return field || typy(metaField, 'value[0]').safeString || ''
```

Comment:
`safeString` should return `''` so you should be able to get away with:
```
return field || typy(metaField, 'value[0]').safeString
```

Reply:
Good call. Updated accordingly.